pkg/minikube/console/console.go

```diff
@@ -0,0 +1,203 @@
+	if err != nil {
+		glog.Errorf("applyStyle(%s): %v", style, err)
+		if oerr := OutLn(format, a...); oerr != nil {
+			glog.Errorf("Out failed: %v", oerr)
+		}
+		return err
+	}
+	return Out(OutStyle)
+}
+
+// Out writes a basic formatted string to stdout
+func Out(format string, a ...interface{}) error {
+	p := message.NewPrinter(preferredLanguage)
+	if outFile == nil {
+		if _, err := p.Fprintf(os.Stdout, "(stdout unset)"+format, a...); err != nil {
+			return err
+		}
+		return fmt.Errorf("no output file has been set")
+	}
+	_, err := p.Fprintf(outFile, format, a...)
+	return err
+}
+
+// OutLn writes a basic formatted string with a newline to stdout
+func OutLn(format string, a ...interface{}) error {
+	return Out(format+"\n", a...)
+}
+
+// ErrStyle writes a stylized and formatted error message to stderr
+func ErrStyle(style, format string, a ...interface{}) error {
+	format, err := applyStyle(style, useColor, fmt.Sprintf(format, a...))
+	if err != nil {
+		glog.Errorf("applyStyle(%s): %v", style, err)
+		if oerr := ErrLn(format, a...); oerr != nil {
+			glog.Errorf("Err(%s) failed: %v", format, oerr)
+		}
+		return err
+	}
+	return Err(format)
+}
+
+// Err writes a basic formatted string to stderr
+func Err(format string, a ...interface{}) error {
+	p := message.NewPrinter(preferredLanguage)
+	if errFile == nil {
+		if _, err := p.Fprintf(os.Stderr, "(stderr unset)"+format, a...); err != nil {
+			return err
+		}
+		return fmt.Errorf("no error file has been set")
+	}
+	_, err := p.Fprintf(errFile, format, a...)
+	return err
+}
+
+// ErrLn writes a basic formatted string with a newline to stderr
+func ErrLn(format string, a ...interface{}) error {
+	return Err(format+"\n", a...)
+}
+
+// Success is a shortcut for writing a styled success message to stdout
+func Success(format string, a ...interface{}) error {
+	return OutStyle("success", format, a...)
+}
+
+// Fatal is a shortcut for writing a styled fatal message to stderr
+func Fatal(format string, a ...interface{}) error {
+	return ErrStyle("fatal", format, a...)
+}
+
+// Warning is a shortcut for writing a styled warning message to stderr
+func Warning(format string, a ...interface{}) error {
+	return ErrStyle("warning", format, a...)
+}
+
+// Failure is a shortcut for writing a styled failure message to stderr
+func Failure(format string, a ...interface{}) error {
+	return ErrStyle("failure", format, a...)
+}
+
+// SetPreferredLanguageTag configures which language future messages should use.
+func SetPreferredLanguageTag(l language.Tag) {
+	glog.Infof("Setting Language to %s ...", l)
+	preferredLanguage = l
+}
+
+// SetPreferredLanguage configures which language future messages should use, based on a LANG string.
+func SetPreferredLanguage(s string) error {
+	if s == "" || s == "C" {
+		SetPreferredLanguageTag(defaultLanguage)
+		return nil
+	}
+	// Ignore encoding preferences: we always output utf8. Handles "de_DE.utf8"
```

Comment:
do we want to warn about when we discard encoding preferences?

Reply:
Clarified comment.